pandas/io/formats/excel.py

```diff
@@ -408,19 +418,7 @@ def __init__(
         self.header = header
         self.merge_cells = merge_cells
         self.inf_rep = inf_rep
-
-    @property
```

Comment:
Why did you change this?

Reply:
To provide an interface to  override (i.e. disable) the cell styling